src/app/signup/signup.component.html

```diff
@@ -2,17 +2,16 @@
   <h1>Volunteer with WAFB today!</h1>
   <mat-form-field class="third-width">
     <mat-label>Full Name</mat-label>
-    <input
-      matInput
-      placeholder="John Smith"
-      [(ngModel)]="name">
+    <input type="text" matInput placeholder="John Smith" [(ngModel)]="name" />
   </mat-form-field>
   <mat-form-field class="third-width">
     <mat-label>Email</mat-label>
     <input
       matInput
+      type="email"
       placeholder="Ex. john@ex.com"
-      [(ngModel)]="email">
+      [(ngModel)]="email"
+    />
```

Comment:
@drkuster Did the same thing here. Make sure that you add the 'type' field to inputs in the future, it's syntactically correct HTML.